src/porepy/utils/grid_utils.py

```diff
@@ -104,3 +106,255 @@ def star_shape_cell_centers(g: "pp.Grid", as_nan: bool = False) -> np.ndarray:
 
     # Shift back the computed cell centers and return them.
     return cell_centers + np.tile(xn_shift, (g.num_cells, 1)).T
+
+
+def compute_circumcenter_2d(sd: pp.TriangleGrid, minimum_angle=np.pi * 0.45):
```

Comment:
I find 'minimum_angle' less than intuitive, considering that this is an upper bound. How about threshold_angle or something similar?